Btcbf.py

```diff
@@ -200,34 +243,51 @@ def get_user_input(self):
             print("Your wallet is ready!")
             input("Press Enter to exit")
             exit()
-        with ThreadPoolExecutor(max_workers=self.num_of_cores()) as pool:
-            r = range(100000000000000000)
-            print("\n Starting ...")
-            self.start_t = time()
+
+        if self.use_process:

```

Comment:
Due to the fact that I don't have enough time to convert all tasks to using the `Process`. I converted the random_brute. That's why I need this flag

Reply:
It will be better if the value is declared in the `if __name__ == "__main__":` and then passed to the class.

Reply:
I want `use_process` to be used for this particular option. The rest should be used the old approach. Then, slowly we can migrate to the new approach.

If we move `use_process` to the outside of the class, how can we switch and meet this expectation?

Reply:
I mean to share the values between `speed()` and the rest.
The problem is that we need to measure the speed and display it. Thus, the current number should be shared to the `speed()` function.

Reply:
I believe that the metrics are shared across the threads. It means that `t1 = threading.Thread(target=obj.speed)` is able to display it correctly.

In details, these lines of code help to to share metrics between `speed()`
```
self.cur_n = self.manager.Value('i', 0)
self.start_t = self.manager.Value('i', 0)
```

Could you specify which information is not shown so that I can look deeper?